nav/load_waypoints/scripts/navigate_waypoints.py

```diff
@@ -148,7 +167,7 @@ def send_and_wait_goal_to_move_base(self, curr_waypoint):
         action_client.send_goal(goal)
 
         # Waits for the server to finish performing the action.
-        finished_within_time = action_client.wait_for_result(rospy.Duration(60))
+        finished_within_time = action_client.wait_for_result(rospy.Duration(6000))
```

Comment:
Just to double check, this returns early if the result is done right? The Duration is just a timeout

Reply:
yes duration is just timeout (in seconds)